src/pages/login.tsx

```diff
@@ -2,24 +2,57 @@ import React, { useState } from "react";
 import type { NextPage } from "next";
 import styled from "@emotion/styled";
 import Link from "next/link";
+import Router from "next/router";
 
 const Login: NextPage = () => {
-  const [idCheck, setIdCheck] = useState("");
-  const [idError, setIdError] = useState(false);
-  const [pwCheck, setPwCheck] = useState("");
-  const [pwError, setPwError] = useState(false);
+  const [userId, setUserId] = useState("");
+  const [userPw, setUserPw] = useState("");
+  const [userIdPwError, setUserIdPwError] = useState(false);
 
+  const handleId = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setUserId(e.currentTarget.value);  
+    console.log(userId); 
+  }
+
+  const handlePassword = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setUserPw(e.currentTarget.value);  
+    console.log(userPw); 
+  }
```

Comment:
p5: custom hook을 사용해서 간편하게 중복적으로 로직을 쓰는 것이 아니라 로직 내부가 같다면 한번에 활용 해볼 수도 있을것 같아요
```suggestion
  const handleId = (e: React.ChangeEvent<HTMLInputElement>) => {
    setUserId(e.currentTarget.value);  
    console.log(userId); 
  }

  const handlePassword = (e: React.ChangeEvent<HTMLInputElement>) => {
    setUserPw(e.currentTarget.value);  
    console.log(userPw); 
  }
```

Reply:
이 부분 관련해서 custom hook 만들어봤는데, 그 부분 참고해서 변경해주시면 로직이 많이 줄 것 같네요!